kibble/models/generic_item.go

```diff
@@ -59,3 +60,26 @@ func (i GenericItem) GetTranslatedTitle(T i18n.TranslateFunc, i18nKey string) st
 	}
 	return i.Title
 }
+
+// get carousel focus
+func (i GenericItem) GetCarouselImageFocusArea() string {
+	switch i.ItemType {
+	case "film":
+		if f, ok := i.InnerItem.(Film); ok {
+			return f.CarouselFocus
+		}
+	case "tvseason":
+		if s, ok := i.InnerItem.(TVSeason); ok {
+			return s.CarouselFocus
+		}
+	case "page":
+		if p, ok := i.InnerItem.(Page); ok {
+			return p.CustomFields.GetString("carousel_focus", "")
+		}
+	case "bundle":
+		if b, ok := i.InnerItem.(Bundle); ok {
+			return b.CustomFields.GetString("carousel_focus", "")
+		}
+	}
+	return i.CarouselFocus
```

Comment:
maybe i should just return empty string here?

Reply:
Yeah it should be an empty string if it can't find one, and I don't think you need to be storing CarouselFocus on GenericItem right? Because this function digs it out of InnerItem.

Reply:
yeah good point. fixed up and tested

Reply:
@PeculiarGoat ready for re-review